libs/core/src/lib/feed-input/feed-input.component.scss

```diff
@@ -4,6 +4,28 @@
     &.is-disabled {
         pointer-events: initial;
         opacity: initial;
+
+        textarea {
+            &::placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &::-webkit-input-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &::-moz-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &:-ms-input-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }
+            &:-moz-placeholder {
+                color: var(--sapTextColor, #32363a);
+            }

```

Comment:
`color: #32363a` should be added everywhere `color: var(--sapTextColor, #32363a);` is used